ansible/roles/ml-data-pipeline/tasks/main.yml

```diff
@@ -0,0 +1,115 @@
+---
+# tasks file for data-pipeline
+- name: Git checkout
+  git:
+    repo: 'https://github.com/Sunbird-Ed/ml-analytics-service.git'
+    dest: "/opt/sparkjobs/ml-analytics-service"
```

Comment:
Can we use the `BASEPATH` variable here?

Reply:
defined the basepath as per suggestion